### ISSUE TITLE
after chosed project endless ... Checking for update.... 

### ISSUE THREAD

**Issue by SebasWW** [Microsoft/WindowsTemplateStudio#2323]
## For Bugs:
after chosed project endless ... Checking for update.... 

Can you give update url  for checking?

### Repro steps
Select Project and Press Ok in Dialog

### Expected Behavior
Continue with wizard

### Actual Behavior
nothing happens

## System
Windows10 Pro

* **VS Version:**
Microsoft Visual Studio Community 2017 
Version 15.7.2
VisualStudio.15.Release/15.7.2+27703.2018
Microsoft .NET Framework
Version 4.7.03056

Installed Version: Community
* **WTS Wizard Version:**

* **WTS Template Version:**
2.1.18124.1
* **Windows Build:**
17134.81

**Comment by sibille** [comment]
Hi, thanks for reporting!
The url is https://wts2.azureedge.net/config.json. Whilst you check that url, I'll have a look into the code, if check for update fails it should extract local templates delivered with the vsix. 

To unblock you and get you started you can launch the wizard without internet connection, it should extract local templates then.

**Comment by SebasWW** [comment]
Yes, i cant connect on port 80. But why updater doesnt throw error?

>ping wts2.azureedge.net.

Обмен пакетами с cs9.wpc.v0cdn.net [152.199.20.1] с 32 байтами данных:
Ответ от 152.199.20.1: число байт=32 время=29мс TTL=58
Ответ от 152.199.20.1: число байт=32 время=34мс TTL=58
Ответ от 152.199.20.1: число байт=32 время=35мс TTL=58
Ответ от 152.199.20.1: число байт=32 время=29мс TTL=58

> telnet wts2.azureedge.net 80
Подключение к wts2.azureedge.net...Не удалось открыть подключение к этому узлу, на порт 80: Connection fails

**Comment by SebasWW** [comment]
ive tried with proxy. it works. problem is ungenerated network error.
i have no problem

**Comment by sibille** [comment]
ok, glad this works now for you!
Reviewing the code I saw the exception is thrown but handled in the wrong place, as if config cannot be downloaded local templates should be extracted. I'm fixing this. 
Thaks for your help!
